platform.md

```diff
@@ -893,6 +899,8 @@ Where:
 
 #### `metadata.toml` (TOML)
 ```toml
+buildpack-default-process-type = "<process type>"
```

Comment:
Do we need the "buildpack-" prefix here? Should we add a top-level key?

Reply:
We wanted `buildpack-` to differentiate between the buildpack-provided default and user-provided default (via the `-process-type` flag). We thought if would be confusing if the app had default process type `some-user-type` and metadata.toml contained `default-process-type = some-buildpack-type`. An alternative that we considered was deleting this key from metadata.toml prior to export, so that it is only used to pass information between phases. 

As for whether it should be a top-level key, I'm not sure... we thought about adding `default = <true or false>` under each buildpack but then we also thought it would be confusing given that multiple buildpacks can each contribute a default and it wouldn't be obvious to the reader of this file that "last wins". 

cc @yaelharel please add if I missed anything!